src/tiledb/cloud/bioimg/ingestion.py

```diff
@@ -219,7 +219,8 @@ def ingest_tiff_udf(
             "png": Converters.PNG,
         }.get(converter, Converters.OMETIFF)
 
-        experimental_reader = kwargs.get("experimental_reader", False)
+        experimental_reader = kwargs.pop("experimental_reader", False)
```

Comment:
@ktsitsi I found that if I didn't pop off this keyword argument, `from_bioimg()` could receive it twice.